frontend/src/components/DefenceBox/DefenceMechanism.css

```diff
@@ -1,106 +1,164 @@
-.defence-mechanism {
-	cursor: default;
+.defence-mechanism-fieldset {
+	min-height: fit-content;
+	margin: 0.1em;
+	border: 0.25rem solid var(--control-config-border);
+	border-radius: 0.5rem;
+}
+
+/* the name of the fieldset */
+.defence-mechanism-legend {
+	padding: 0.3em;
```

Comment:
the article where I found the accessible toggle button says this about ems:
"All sizing is done in ems so these will scale with the surrounding text."

Reply:
Curious. I guess it makes sense that you want the toggle to match the size of the text that it is near.

The difference I can find is that `1 rem` is a size relative to the root font size and `em` is relative to the the font size of the element that you're styling (whether set explicitly or inherited from an ancestor). So in this case if we changed the size of the legend text... it will scale the padding accordingly. Which is great!

Consider my first comment nullified :)